### PR TITLE
[ADD][13] sale_optional_product_section

### PR DESCRIPTION
**Pull request by TDu**

### CODE REVIEW

File: sale_optional_product_section/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* `Camptocamp <https://www.camptocamp.com>`_:
+
+  * Thierry Ducrest
```
Comment (i-vyshnevska) [inline sale_optional_product_section/readme/CONTRIBUTORS.rst:3]:
```suggestion
* Thierry Ducres <....@camptocamp.com>
```

**Review by i-vyshnevska — APPROVED** [review]

**Review by sebalix — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: LGTM

File: sale_optional_product_section/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{
+    "name": "Sale Optional Product Sections",
+    "summary": "Add sections in the optional product list of a sale order",
+    "version": "13.0.0.1.0",
+    "author": "Camptocamp,Odoo Community Association (OCA)",
+    "category": "Sale",
+    "website": "http://github.com/OCA/sale-reporting",
+    "license": "LGPL-3",
```
Comment (rousseldenis) [inline sale_optional_product_section/__manifest__.py]: Is it an AGPL or LPGL one ? licensing marks are opposite through the file

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Comment by rousseldenis** [comment]
@TDu Could you fix this ?

**Comment by TDu** [comment]
@rousseldenis I rebased and fixed the linting. I think the licence is also good now ?

**Pull request closed without merging**